Sources/SwiftSyntax/SourceLocation.swift

```diff
@@ -43,15 +59,22 @@ public struct SourceLocation: Hashable, Codable, CustomDebugStringConvertible {
 public struct SourceRange: Hashable, Codable, CustomDebugStringConvertible {
 
   /// The beginning location in the source range.
+  ///
+  /// This location is included in the range
   public let start: SourceLocation
 
   /// The beginning location in the source range.
```

Comment:
`beginning` -> `end`. `in the` also seems weird`, maybe `of the` instead (for both this and above)? It'd be nice to use `half-open` here somewhere, possibly in the documentation for the type itself.

```
This location is no longer part of the range
```
How about:
```
The location of the character after the end of the range, ie. this location is not included in the range.
```